### DISCUSSION TITLE
Useful analyses for presocial0.1

### DISCUSSION THREAD

**Discussion by jkbhagatio** [SainsburyWellcomeCentre/aeon_analysis#2] [General]
- Which statistical test(s) to use to compare distributions for easy and hard patch cumulative wheel distance?
    - Wilcoxon rank-sum?
- When doing stats and viz: 
    - Look at cumulative wheel distance data post threshold change.
    - For a particular lambda, look at:
        - all sessions together
        - all mice for a particular session number together
        - all sessions for a particular mouse together

**Comment by jerlich** [comment]
I think we did something like  compute ` delta_prefA = log2 ((postA/preA) / (postB/preB))` where pre and post are the wheel distance in some time interval (30 min or 1 hour) before and after a change. If there was no change you expect this to be normal with mean 0.  

then for aggregate stats you can do a mixed effect model like `delta_prefA ~ delta_threshold + (delta_threshold | subjid)` so  you can get an estimate of the sensitivity of each mouse to the change.  (or just do `delta_prefA ~ delta_threshold` separately for each mouse. You can use [this code](https://www.pythonfordatascience.org/mixed-effects-regression-python/#test_with_python) if you want to use python. Otherwise lme4 in R or MixedModels.jl
